### ISSUE TITLE
All orderhistory data listed as LimitBuy

### ISSUE THREAD

**Issue by handyface** [JKorf/Bittrex.Net#70]
Hi!

When I use client.GetOrderHistory all of my order results have an OrderSide of LimitBuy. I'm using 1.3.15.

```C#
var orderHistory = client.GetOrderHistory();
bool hasLimitSell = orderHistory.Result.Any(o => o.OrderSide == OrderSideExtended.LimitSell); // Always false
```

When I check in Bittrex, a lot of the orders listed as LimitBuy were actually a sell order. Anyone else affected by this?

**Comment by JKorf** [comment]
Hi, this was a bug, it is fixed in the latest version (1.4.0) being pushed now.

**Comment by handyface** [comment]
Thanks, I appreciate your work!
